cli/tests/init_module_src_path.sh

```
@@ -0,0 +1,29 @@
+if [ -d "output/init-module" ]; then
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)